integrations/docker/images/vscode/chip-build-vscode/Dockerfile

```diff
@@ -142,6 +133,7 @@ ENV ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
 ENV ZEPHYR_NXP_BASE=/opt/nxp-zephyr/zephyrproject/zephyr
 ENV ZEPHYR_NXP_SDK_INSTALL_DIR=/opt/nxp-zephyr/zephyr-sdk-0.16.8
 ENV NXP_UPDATE_SDK_SCRIPT_DOCKER=/opt/nxp/nxp_matter_support/scripts/update_nxp_sdk.py
+ENV NXP_SDK_PATH=/opt/nxp
```

Comment:
This does not seem to be a env set in chip-build-nxp ... should the env variables be in sync?